lidar_model : "OS" # OS or HDL
cloud_info_topic: "/lvi_sam/lidar/feature/cloud_info"
odom_topic: "odometry/imu"
ptCloudMsg_topic: "/lig_node/GSeged_node"
sensor_height : 1.5   #HUSKY


nodegen:
    node_topic: "/lig_node/pre_processing_node"
    local_frame: "base_link"
    lidar_frame: "os_sensor"
    ptCloud_type: "GAlign"
    lidar_align_frame: "sensor_align"

    min_range: 2.0
    max_range: 80.0
    window_size: 5


patchwork:
    mode: "non_uniform" # [uniform or non_uniform]
    use_z_thr: true
    tuning_mode: true
    reject_reflection_error: false
    visualize: true

    num_iter: 3
    num_lpr: 20
    num_min_pts: 5
    th_seeds: 0.5
    th_dist: 0.125
    max_r: 20.0
    min_r: 2.0
    num_rings: 16
    num_sectors: 54
    normal_thr: 0.707 # 45: 0.707 / 60: 0.866
    obstacle_height_thr: 1.5
    #HUSKY
    adaptive_seed_selection_margin: -2.5
    Z_THR_: [-0.2, -0.0, 0.2, +0.4]  # For elevation origin parameter
    SV_THR_: [0.0, 0.000125, 0.000185, 0.000185] # For flatness
    

tMap:
    msg_topic: "patchwork_ptClouds"
    max_range: 20.0
    min_range: 2.0
    map_resolution: 0.2
    ptCloud_nonground_num_: 50
    odom_diff_threshold_ : 1.0
    map_window_size : 100
    margin_node: false
    updateMap: true